recipes/labjackpython/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "LabJackPython" %}
+{% set org = "labjack" %}
+{% set upstreamversion = "4-24-2014" %}
+{% set version = "20140424" %}
```

Comment:
Would be good if we could determine one of these programmatically from the other. That way there is less chance of a having a mismatch in a release.